src/wallet/coincontrol.h

```diff
@@ -27,6 +27,8 @@ class CCoinControl
     boost::optional<CFeeRate> m_feerate;
     //! Override the default confirmation target if set
     boost::optional<unsigned int> m_confirm_target;
+    //! Avoid partial use of funds sent to a given address: -1 means default
+    short m_avoid_partial_spends;
```

Comment:
I want to avoid using a `bool` here because it's important to distinguish `true`/`false` from 'unset', which should use `DEFAULT_AVOIDPARTIALSPENDS`.